web/app/migrations/0035_auto_20200521_1555.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.10 on 2020-05-21 15:55
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0034_auto_20200311_1805'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PrintShotFeedback',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('image_url', models.CharField(max_length=2000)),
+                ('answer', models.BooleanField(choices=[(True, 'It contains spaghetti'), (False, 'It does NOT contain spaghetti'), (None, "I'll decide later")], null=True)),
```

Comment:
Let's make it a `CharField`. We may need to extend it to accommodate more choices